### ISSUE TITLE
Structured option defaults and examples are rendered as JSON

### ISSUE THREAD

**Issue by lheckemann** [NixOS/nixos-search#233]
```
10:13 <Reventlov> hey there
10:13 <Reventlov> why ther are commas in https://search.nixos.org/options?channel=20.09&show=services.unbound.interfaces&from=0&size=30&sort=relevance&query=services.unbound.interfaces ?
10:13 <Reventlov> i.e. Default value ["127.0.0.1", "::1"]
```
https://search.nixos.org/options?channel=20.09&show=services.unbound.interfaces&from=0&size=30&sort=relevance&query=services.unbound.interfaces

**Comment by turboMaCk** [comment]
@garbas seems like something you should make call on yourself

**Comment by garbas** [comment]
yes, this is a duplicate of #175 and #161 and probably some more. I never got around to fix it.

**Comment by ncfavier** [comment]
This was fixed in #239.

**Comment by cgevans** [comment]
I'm seeing this again.  Has there been a regression here?  Eg, [services.openssh.listenAddresses](https://search.nixos.org/options?channel=21.05&show=services.openssh.listenAddresses&from=0&size=50&sort=relevance&type=packages&query=openssh) has example

```
[
  {
    "addr": "192.168.3.1",
    "port": 22
  },
  {
    "addr": "0.0.0.0",
    "port": 64022
  }
]
```

**Comment by ncfavier** [comment]
cc @ysndr #320

**Comment by ysndr** [comment]
Ill look into that, There seems to be conversion to JSON done by nix somewhere in between

**Comment by ysndr** [comment]
Fixed in #357
